app/Http/Controllers/SprintSnapshotsController.php

```diff
@@ -54,7 +54,7 @@ public function delete(SprintSnapshot $snapshot)
 
 	private function getSprintDataFactory(SprintSnapshot $snapshot)
 	{
-		$sprintData = json_decode($snapshot->data, true);
+		$sprintData = json_decode($snapshot->getData(), true);
```

Comment:
Is that really needed here? I think in the SprintSnapshotsController the full snapshot data will be loaded anyways. Am I wrong?